web/packages/teleport/src/Integrations/Enroll/AwsConsole/IamIntegration/IamIntegration.tsx

```diff
@@ -0,0 +1,434 @@
+import { TextSelectCopyMulti } from 'shared/components/TextSelectCopy';
+import Validation, { Validator } from 'shared/components/Validation';
+import { requiredField } from 'shared/components/Validation/rules';
+
+import { EmptyState } from 'teleport/Bots/List/EmptyState/EmptyState';
+import { FeatureBox } from 'teleport/components/Layout';
+import cfg from 'teleport/config';
+import { Guide } from 'teleport/Integrations/Enroll/AwsConsole/IamIntegration/Guide';
+import { validTrustAnchorInput } from 'teleport/Integrations/Enroll/AwsConsole/rules';
+import {
+  cloudShell,
+  rolesAnywhere,
+} from 'teleport/Integrations/Enroll/awsLinks';
+import { useAwsOidcStatus } from 'teleport/Integrations/status/AwsOidc/useAwsOidcStatus';
+import {
+  AwsRolesAnywherePingResponse,
+  IntegrationKind,
+  integrationService,
+} from 'teleport/services/integrations';
+import useTeleport from 'teleport/useTeleport';
+
+enum Phase {
+  One, // enable step one
+  Two, // enable step two & three
+  Three, // step three is verified, enable proceed button
+}
+
+export function IamIntegration() {
+  const ctx = useTeleport();
+  const flags = ctx.getFeatureFlags();
+  const canEnroll = flags.enrollIntegrations;
+  const clusterId = ctx.storeUser.getClusterId();
+  const resourceRoute = cfg.getUnifiedResourcesRoute(clusterId);
+  const { statsAttempt } = useAwsOidcStatus();
+
+  const { name } = useParams<{ name: string }>();
+  const history = useHistory();
+  const [phase, setPhase] = useState(Phase.One);
+  const [alert, setAlert] = useState<ReactNode>();
+  const [scriptUrl, setScriptUrl] = useState<string>();
+  const [sync, setSync] = useState<string>();
+  const [output, setOutput] = useState<{
+    trustAnchorArn: string;
+    syncRoleArn: string;
+    syncProfileArn: string;
+  }>();
+
+  const [formState, setFormState] = useState({
+    // integrationName: the name of the AWS IAM Roles Anywhere Integration
+    // required by user, no default
+    integrationName: '',
+    // trustAnchor: the name of the Trust Anchor to be created
+    trustAnchorName: 'teleport-aws-roles-anywhere-trust-anchor',
+    // syncRole: the name of the IAM Role to be created
+    syncProfileName: 'teleport-aws-roles-anywhere-profile',
+    // syncProfile: the name of the IAM Roles Anywhere Profile to be created
+    syncRoleName: 'teleport-aws-roles-anywhere-role',
+  });
+
+  function generateCommand(validator: Validator) {
+    if (!validator.validate()) {
+      return;
+    }
+
+    setScriptUrl(
+      cfg.getAwsRolesAnywhereGenerateUrl(
+        formState.integrationName,
+        formState.trustAnchorName,
+        formState.syncRoleName,
+        formState.syncProfileName
+      )
+    );
+    setPhase(Phase.Two);
+  }
+
+  const manageAction = {
+    content: (
+      <Link href={rolesAnywhere} target="_blank">
+        Manage AWS Roles Anywhere Profiles
+        <NewTab size={18} ml={2} />
+      </Link>
```

Comment:
the text and icon needs to be centered
<img width="338" height="48" alt="image" src="https://github.com/user-attachments/assets/eb3ba0ae-100c-4b49-9fb8-2379f0322fe6" />